packages/react-instantsearch/src/core/InstantSearch.test.js

```diff
@@ -255,6 +258,64 @@ describe('InstantSearch', () => {
     expect(ism.skipSearch.mock.calls).toHaveLength(1);
   });
 
+  it('refreshes the cache when the refresh prop is set to true', () => {
+    const ism = {
+      clearCache: jest.fn(),
+    };
+
+    createInstantSearchManager.mockImplementation(() => ism);
+
+    const wrapper = mount(
```

Comment:
You could probably use `shallow` instead of `mount`. The former will only create a tree for the root level (much better for more focused unit test), the latter will perform a full rendering of the DOM. In you test you don't need to perform the full rendering.